src/planner/Planner.cpp

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "planner/Planner.h"
+
+#include "validator/Validator.h"
+
+namespace nebula {
+namespace graph {
+
+std::unordered_map<Sentence::Kind, std::vector<Planner*>> Planner::plannersMap_;
```

Comment:
Is it better to manage the Planners by tree or DAG?

Reply:
Emm, the planners don't have that kind of relation.